app/views/trainees/outcome_details/confirm.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render PageTitle::View.new(title: "trainees.outcome_details.confirm") %>
+<%= render PageTitle::View.new(title: "trainees.outcome_details.confirm_#{@trainee.get_award_type.downcase}") %>
```

Comment:
A little picky! But it might be nicer to read `@trainee.award_type`.

Reply:
All done 👍